numpy/lib/arraysetops.py

```diff
@@ -411,6 +417,8 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
 
     See Also
     --------
+    isin                  : Version of this function that preserves the
+                            shape of ar1.
     numpy.lib.arraysetops : Module with a number of other functions for
```

Comment:
"See also: The module you're looking at right now" is a little weird. @brsr, you don't need to worry about it in this PR though.